src/Acme/Bundle/RangeBundle/Model/RangeProductValue.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Acme\Bundle\RangeBundle\Model;
+
+use Pim\Component\Catalog\Model\AbstractProductValue;
+use Pim\Component\Catalog\Model\ProductValueInterface;
+
+/**
+ * A product value that can only contains range data.
+ *
+ * @author    Damien Carcel (damien.carcel@akeneo.com)
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+class RangeProductValue extends AbstractProductValue implements ProductValueInterface
```

Comment:
I could have made my own product value, but I would had problem with the interface, so it was simpler this way.